game-core/src/main/java/games/strategy/triplea/delegate/battle/BattleActions.java

```diff
@@ -39,6 +39,10 @@ void landParatroopers(
 
   void attackerWins(IDelegateBridge bridge);
 
+  void defenderWins(IDelegateBridge bridge);
+
+  void nobodyWins(IDelegateBridge bridge);
```

Comment:
The "xxWins" seems to be an enumeration of the ' WhoWon' enum. WDYT if we parameterized the API instead?

EG:
```
battleComplete(IDelegateBridge bridge,  WhoWon battleResult);
```

Reply:
Yup, one of the things I plan to work on after this is done.